NMLocalizedPhoneCountryView/NMLocalizedPhoneCountryViewController.swift

```diff
@@ -268,6 +268,7 @@ extension NMLocalizedPhoneCountryViewController: UISearchResultsUpdating {
 extension NMLocalizedPhoneCountryViewController: UISearchBarDelegate {
     func searchBarTextDidBeginEditing(_ searchBar: UISearchBar) {
         // Hide the back/left navigationItem button
+        navigationItem.leftBarButtonItem = nil
```

Comment:
why this?

Reply:
because we don't want user to go back while editing search bar. It will keep the search bar on the previous screen when user go back

Reply:
but if a viewController is pushed, user can use pan gesture to go back. 
If that gesture is still available when `leftBarButtonItem` is nil then user can still go back.
Please verify this.

Reply:
this statement was there in library https://github.com/namshi/NMLocalizedPhoneCountryView/commit/4083fec6db70ec06c92c7d3b3da15d6d5c5f2aed#diff-88fe5f8b8791ca2cf0aac26ce94155ceL265. We removed it because we dont want the search bar to hide navigation bar while editing. but this will show back button which is a problem to show

Reply:
ok